README.md

```diff
@@ -1,2 +1,246 @@
 # deploio-landing-page
 This is the landing page for deplo.io.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Requirements](#requirements)
+- [Getting Started](#getting-started)
+- [Important Note](#important-note)
+- [Testing](#testing)
+- [Preparing for Deployment](#preparing-for-deployment)
+- [DevTools](#devtools)
+- [Installed Packages](#installed-packages)
+
+## Overview
+
+Built with type safety, scalability, and developer experience in mind. A batteries included Vite + React template.
+
+- [TypeScript](https://www.typescriptlang.org) - A typed superset of JavaScript designed with large scale applications in mind
+- [ESLint](https://eslint.org) - Static code analysis to help find problems within a codebase
+- [Prettier](https://prettier.io) - An opinionated code formatter
+- [Vite](https://vitejs.dev) - Feature rich and highly optimized frontend tooling with TypeScript support out of the box
+- [React](https://react.dev) - A modern front-end JavaScript library for building user interfaces based on components
+- [Tailwind CSS](https://tailwindcss.com) - A utility-first CSS framework packed with classes to build any web design imaginable
+- [Storybook](https://storybook.js.org) - A frontend workshop for building UI components and pages in isolation
+- [TanStack Query](https://tanstack.com/query/latest) - Declarative, always-up-to-date auto-managed queries and mutations
+- [Zustand](https://zustand-demo.pmnd.rs) - An unopinionated, small, fast and scalable bearbones state-management solution
+- [React Hook Form](https://react-hook-form.com) - Performant, flexible and extensible forms with easy-to-use validation
+- [React Testing Library](https://testing-library.com) - A very light-weight, best practice first, solution for testing React components
+- [@swc/jest](https://swc.rs/docs/usage/jest) - To make your Jest tests run faster, you can swap out the default JavaScript-based runner
+- [Faker](https://fakerjs.dev/) - Generate massive amounts of fake (but realistic) data for testing and development
+- [Husky](https://github.com/typicode/husky#readme) + [Commitizen](https://github.com/commitizen/cz-cli#readme) + [Commitlint](https://github.com/conventional-changelog/commitlint#readme) - Git hooks and commit linting to ensure use of descriptive and practical commit messages
+- [ts-reset](https://github.com/total-typescript/ts-reset#readme) - Improvements for TypeScripts built-in typings for use in applications
+
+A more detailed list of the included packages can be found in the [Installed Packages](#installed-packages) section. Packages not shown above include Devtools, ui helper libraries, and eslint plugins/configs.
+
+## Requirements
+
+- [NodeJS 20+](https://nodejs.org/en)
+- [npm](https://npm.io) (or equivalent)
+
+## Getting Started
+
+Getting started is a simple as cloning the repository
+
+```
+git clone git@github.com:ninech/pricing-calculator.git
+```
+
+Changing into the new directory
+
+```
+cd pricing-calculator
```

Comment:
You are right, I have to rewrite the README.md so that it better suits the deploio-site. This README has been taken from the pricing-calculator repository.